### ISSUE TITLE
Documentation: leftovers on https://ammonite.io from before removal of ammonite-shell

### ISSUE THREAD

**Issue by rom1dep** [com-lihaoyi/Ammonite#1257]
Searching the page still brings up occurrences of removed patterns from ammonite-shell, like `ls!` (9 hits) `||` (7 hits)

**Comment by dcm** [comment]
Yes, would be nice if these were removed from the docs. I installed Ammonite a few months ago, and like once a week I'll try to use if for something --> I try shell commands --> they don't work --> I think "well I guess it's broken" --> I stop using it.

**Comment by alexarchambault** [comment]
These need to be updated, yes. Feel free to submit PRs to remove those.
